tideways.c

```diff
@@ -2532,6 +2532,7 @@ void hp_init_trace_callbacks(TSRMLS_D)
 
     cb = tw_trace_callback_event_dispatchers2;
     register_trace_callback("HookCore::coreCallHook", cb); // PrestaShop 1.6
+    register_trace_callback("Neos\\Flow\\SignalSlot\\Dispatcher::dispatch", cb);
```

Comment:
In theory that should also need the `_Original` suffix, I just checked and we build a proxy class for the `Dispatcher`. It could very well be I never spotted this missing, since I don't know it would exist in a trace and/or how it would look.